thanos/README.md

```diff
@@ -122,6 +122,41 @@ spec:
   releaseNamespace: kube-monitoring
 ```
 
+### Thanos Ruler
+To enable Thanos Ruler component creation (Thanos Ruler is disabled by default)
+
+```yaml
+spec:
+  optionsValues:
+  - name: thanos.ruler.enabled
+    value: true
+```
+
+#### Configuration
+##### Alertmanager
+For Thanos Ruler to communicate with Alertmanager we need to enbale the approprite configuration and provide Plugin with secret/key names containing
+necessary SSO key and certificate.
+
+eg. of Plugin setup with Thanos Ruler using Alertmanager
+```yaml
+spec:
+  optionsValues:
+  - name: thanos.ruler.enabled
+    value: true
+  - name: thanos.ruler.alertmanagers.enabled
+    value: true
+  - name: thanos.ruler.alertmanagers.authentication.ssoCert
+    valueFrom:
+      secret:
+        key: <keyName>
+        name: <secretName>
+  - name: thanos.ruler.alertmanagers.authentication.ssoKey
+    valueFrom:
+      secret:
+        key: <keyName>
+        name: <secretName>
```

Comment:
Please use $KEY_NAME and $SECRET_NAME to adapt to the style of the rest of this readme.